### PR TITLE
Ua4i consulting

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Build the pricing/consulting page
- Add to the navigation

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /pricing/consulting
- Check that is matches the [copy doc](https://docs.google.com/document/d/1sXeOQ9eLq53NcfbrorYstHkZYTdut33jbN5LyHoDdYY/edit#)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5012

### CODE REVIEW

File: navigation.yaml [hunk 1]
```diff
@@ -643,3 +643,5 @@ pricing:
       path: /pricing
     - title: Ubuntu Advantage Infrastructure
```
Comment (pmahnke) [inline navigation.yaml]: Can you make this Support for now? see the Readme in the folder (link on the spreadsheet)

File: templates/pricing/consulting.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        <p>Co-design workshops determine the optimal architecture for your workloads and integration requirements. Our engineers then deploy it for you.</p>
+        <p>What&rsquo;s included:</p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Workload analysis</li>
+          <li class="p-list__item is-ticked">VMware migration plan</li>
+          <li class="p-list__item is-ticked">Customised architecture</li>
+          <li class="p-list__item is-ticked">Containerised or isolated control plane</li>
+          <li class="p-list__item is-ticked">Storage and SDN integration</li>
+          <li class="p-list__item is-ticked">LDAP or Active Directory</li>
+          <li class="p-list__item is-ticked">Corporate monitoring</li>`
```
Comment (pmahnke) [inline templates/pricing/consulting.html]: you have a funny character at the end here that is creating a newline

**Comment by anasereijo** [comment]
@anthonydillon 
on 'Private Cloud Build Plus' please delete that character and the paragraph
Consulting   Ubuntu

File: navigation.yaml (hunk 1 above)
Comment (anthonydillon) [reply navigation.yaml]: Can't find a comment about this? Did you still want the URL to be /pricing/infra?

**Comment by anasereijo** [comment]
👍 LGTM

**Pull request merged**